pgrx-pg-sys/include/pg12.h

```diff
@@ -167,6 +168,7 @@
 #include "utils/regproc.h"
 #include "utils/rel.h"
 #include "utils/relcache.h"
+#include "utils/resowner_private.h"
```

Comment:
why is this one private?

Reply:
I have no idea. It claims to be private but it contains functions which are necessary for anyone implementing a JIT provider. It might be an error that these functions ended up in this header file.

Reply:
And the contents of it does not look very private.

Reply:
This commit split the header file: https://github.com/postgres/postgres/commit/45326c5a11530cff5db99c6d4b393439cc901f26

Reply:
Let's include resowner.h then also, so we can just pull a line on the pg17 upgrade.

Reply:
I have done so now.